### PR TITLE
Add remote option to job 

### PR DESCRIPTION
**Pull request by maartenvanvliet**
* Adds the remote field to the jobs table
* Shows remote option for jobs in company card
* Adds remote option to create/update form

### CODE REVIEW

File: lib/companies/schema/job.ex [hunk 1]
```diff
@@ -7,6 +7,7 @@ defmodule Companies.Schema.Job do
   schema "jobs" do
     field :title, :string
     field :url, :string
+    field :remote, :boolean
```
Comment (pdgonzalez872) [inline lib/companies/schema/job.ex]: Nit: What about when a job is not just remote or not? What if it is "remote for the right candidate"?

Comment (doomspork) [reply lib/companies/schema/job.ex]: That's a good question @pdgonzalez872! I think a note could go in the job description. On other job sites I've seen those jobs listed as on-site with a note about remote for the right person.

Thoughts @maartenvanvliet? @gemantzu?

Comment (maartenvanvliet) [reply lib/companies/schema/job.ex]: don't think it needs to be covered right now, as said a note could go in a description.

Comment (gemantzu) [reply lib/companies/schema/job.ex]: @doomspork I think you are right, we can not cover all cases like that, but as an afterthought we could turn it into a dropdown selection ("remote", "on-site", "remote for the right person") as a new Schema (JobLocation), and we could keep adding to it as needed. 

But I believe this is not actually needed, simply adding a remote field should suffice for 99% of the use cases.

**Comment by doomspork** [comment]
@maartenvanvliet thank you so much! There are plenty of improvements that could be made to the site, we'd more than welcome your assistance 😁

**Comment by maartenvanvliet** [comment]
👍

**Pull request merged**